api/server/server.go

```diff
@@ -594,9 +594,19 @@ func (s *Server) getContainersLogs(version version.Version, w http.ResponseWrite
 		return fmt.Errorf("Bad parameters: you must choose at least one stream")
 	}
 
+	var since time.Time
+	if r.Form.Get("since") != "" {
+		s, err := strconv.ParseInt(r.Form.Get("since"), 10, 64)
+		if err != nil {
+			return err
+		}
+		since = time.Unix(s, 0)
```

Comment:
@duglin other usages in this file are either (1) converting to time.Time at some point later on or (2) the "since" is not a time.

I preferred converting and storing since to time.Time right here so that it doesn't get passed to stack below as int and has a special case s==0.